specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,86 @@
+# Usage Unit
+
+Usage Unit refers to a unit of measurement for the consumption of resources or services. The Usage Unit for a particular line item may differ from the published Pricing Unit when providers use different units, unit increments, or dimensions for calculating cost.  This unit is commonly used when auditing or reconciling consumption data.
```

Comment:
At this point I don't have a suggestion for improvement :(, but this reference/comparison to Pricing Unit seems a bit unclear. It implies that there are UCs in which providers use different columns for cost calculation? The prescribed calculation is QuantityInPricingUnit multiplied by the corresponding unit price.

Reply:
I know, right?  I revised it form a previous version.  I can see the reasoning - I know of one provider that uses different "usage" units than "pricing" units.   Let's seek some input form @arrarama and @flanakin here

Reply:
Discussed in the VS3 meeting with a number of folks - I'm going to add language to make it crystal clear that this shouldn't be used for pricing calc

Reply:
I'll open up another discussion against the commits I just made to add in one clarifying sentence.